src/main/java/mate/academy/Main.java

```diff
@@ -2,15 +2,13 @@
 
 import java.util.concurrent.CompletableFuture;
 import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
 
 public class Main {
     public static void main(String[] args) {
-        // Feel free to play with AsyncRequestProcessor in this main method if you want
-        ExecutorService executor = null; // Provide implementation that fits your needs
+        ExecutorService executor = Executors.newFixedThreadPool(5);
         AsyncRequestProcessor asyncRequestProcessor = new AsyncRequestProcessor(executor);
-
-        // Simulating multiple concurrent requests
-        String[] userIds = {"user1", "user2", "user3", "user1"}; // Note: "user1" is repeated
+        String[] userIds = {"user1", "user2", "user3", "user1"};
```

Comment:
Using an array of user IDs where the same ID can appear multiple times might lead to redundant processing. If processing is idempotent and costly, consider removing duplicates before processing.